lobby/lobby.js

```diff
@@ -1,55 +1,58 @@
-const dgram = require('dgram');
-const NetCode = require('../utils/NetCode.js');
-const OpCode = require('../utils/Commands.js').OpCode;
-const Game = require('./Game.js');
+const dgram = require("dgram")
+const OpCode = require("../utils/Commands.js").OpCode
+const Game = require("./Game.js")
 
-const PORT = process.argv[2]; // 'node lobby.js port'
+const PORT = process.argv[2] // 'node lobby.js port'
 
-const server = dgram.createSocket('udp4');
+const server = dgram.createSocket("udp4")
 
-const game = new Game(server, process);
+const game = new Game(server, process)
 
-server.on('error', (err) => {
-    console.log(`error:\n${err.stack}`);
-    server.close();
-    process.exit();
-});
+server.on("error", (err) => {
+    console.log(`error:\n${err.stack}`)
+    server.close()
+    process.exit()
+})
 
-server.on('message', (data, client) => {
-    let receivedCmd = new DataView(data.buffer);
-    if(receivedCmd.byteLength <= 0) return;
+server.on("message", (data, client) => {
+    let receivedCmd = new DataView(data.buffer)
+    if(receivedCmd.byteLength <= 0) return
 
-    let op = receivedCmd.getInt8(0);
-    console.log(`got ${OpCode.ToString(op)} from ${client.address}:${client.port}`);
+    let op = receivedCmd.getInt8(0)
+    console.log(`got ${OpCode.ToString(op)} from ${client.address}:${client.port}`)
 
     switch(op){
         case OpCode.Register:
-            game.Connect(client);
-            game.Spawn(client.port);
-            break;
-        case OpCode.Jump:
-            let jumpHeight = receivedCmd.getFloat32(1, true);
-            game.Jump(client.port, jumpHeight);
-            break;
-        case OpCode.Move:
-            let moveSpeed = receivedCmd.getFloat32(1, true);
-            game.Move(client.port, moveSpeed);
-            break;
-        case OpCode.Disconnect:
-            game.Disconnect(client.port);
-            break;
-        case OpCode.Ping:
-            game.Pong(client.port);
-            break;
+            game.Connect(client)
+            game.Spawn(client.port)
+            break
+        case OpCode.Jump:{
```

Comment:
Eslint is enforcing **no space** before { } because it's a block.
But i really would like a space after a _case X:_

There's _switch-colon-spacing_ to enforce a space after the _case_ but it conflicts with _space-before-blocks_ rule since this is still a { } block.
I think i'll have to live with this.